### DISCUSSION TITLE
A bug in "Override Items" in the AdGuardHome.

### DISCUSSION THREAD

**Discussion by louis5974** [AdguardTeam/AdGuardHome#4384] [Q&A]
@ainar-g 
![微信截图_20220311091504](https://user-images.githubusercontent.com/53338257/157783376-6ff25a27-30da-43d9-bff8-8194901dd215.png)
![微信截图_20220311091754](https://user-images.githubusercontent.com/53338257/157783384-06b9fadf-f736-4133-ad95-c6e3f5183d0d.png)

【*.example.org–  Rewrite example.org  responses for all subdomains.】
for example：
I want to set override:   *.speedtest.net

When my computer's DNS cache is flushed.
ping speedtest.net , it doesn't work. I have tested " *.example.org " with this problem.
The picture shows the place selected by the red frame. 
My version is AdGuardHome  v0.107.5 
Hopefully this will be fixed.
Thank Ainar Garipov ！

**Comment by louis5974** [comment]
Responses for all subdomains cannot be rewritten.
I can only fill in the rewrite items for each individual domain name.

**Reply by louis5974** [reply]
@ainar-g

**Reply by louis5974** [reply]
@ainar-g

**Comment by louis5974** [comment]
@ainar-g

**Comment by ainar-g** [comment]
@louis5974, apologies for the late response.  Something that looks like a bug is better off in the Issues.

Is this the only rewrite you have?  Perhaps it interacts badly with some other rewrites?

**Reply by louis5974** [reply]
Hi Ainar Garipov. I finally waited for your reply. 
The problem shown in my screenshot, I can confirm that there is only one override item. You can try your situation and use "*.example.org" in the rewrite item to invalidate. My English is not good, I am Chinese, please forgive me if there is a problem of communication.

You can test it on your computer and see if there is a problem.

**Reply by louis5974** [reply]
I currently have about 80 rewrite items in the system. I can confirm that there is no correlation with "*.speedtest.net", nothing else.

**Reply by ainar-g** [reply]
@louis5974, have you tried adding a rule for `speedtest.net`, without the `*.`, as well?  As the input text says, `*.speedtest.net` would only rewrite subdomains of `speedtest.net`, such as `www.speedtest.net`.
